docs/sources/explore/index.md

```diff
@@ -49,9 +49,15 @@ In split view, timepickers for both panels can be linked (if you change one, the
 
 You can close the newly created query by clicking on the Close Split button.
 
+## Share shortened link
+
+> Share shortened link is only available in Grafana 7.3 and above.
+
+The Share shortened link capability allows you to create smaller and simpler URLs of the format /goto/:uid instead of using longer URLs containing complex query parameters. You can create a shortened link by clicking on the **Share** option in Explore toolbar.
```

Comment:
```suggestion
The Share shortened link feature allows you to create smaller and simpler URLs of the format /goto/:uid instead of using longer URLs containing complex query parameters. You can create a shortened link by clicking on the **Share** option in Explore toolbar.
```
does it sound better?

Reply:
This was actually changed by @achatterjee-grafana , our technical write. Both options sounds good to me.

Reply:
Hi @Elfo404, "capability" is a better word here:

You can read up for as to why: https://www.quora.com/What-is-the-difference-between-features-and-capabilities#:~:text=A%20capability%20is%20the%20ability,system%20and%20makes%20features%20possible.&text=of%20the%20term.-,A%20feature%20is%20something%20that%20the%20system%20does%20that%20is,associated%20with%20the%20user%20interface.

Reply:
@achatterjee-grafana Got ya! Thanks for the link! :)